amgut/lib/data_access/test/test_ag_data_access.py

```diff
@@ -17,7 +17,13 @@ def tearDown(self):
         del self.ag_data
 
     def test_authenticateWebAppUser(self):
```

Comment:
can you add a test please for a kit ID that is assured to not exist in the system? it would be great to cover both with an password that doesn't exist as well as one that does (i.e., "test")